index.html

```diff
@@ -0,0 +1,343 @@
+                <button type="submit" class="dropdown">
+                  <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+                </button>
+              </li>
+              <li>
+                <ul class="language-list">
+                  <li class="JavaScript1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-1.png" alt="" />
+                      <p>JavaSript</p>
+                    </div>
+                  </li>
+                  <li class="HTML1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-2.png" alt="" />
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li class="CSS1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-3.png" alt="" />
+                      <p>CSS</p>
+                    </div>
+                  </li>
+                </ul>
+              </li>
+            </li>
+            <ul class="event-skill">
+              <li class="event18">
+                <h3>Frameworks</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+              <li class="event18">
+                <h3>Skills</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="consection">
+      <ul class="contain">
+        <li class="contact">
+          <h1 >Contact me</h1>
+          <p>
+            If you have an application you</br>
+            are interested in developing, a</br>
+            feature that you need built or a</br>
+            project that needs coding.</br>
+            I’d love to help with it!            
+          </p>
+          <form action="https://formspree.io/f/xayzkwne" class="form" method="post">
+            <li class="form-group">              
+              <input class="types"  type="text" id="name" placeholder="John Vasquez" maxlength="30" required/>
+            </li>
+            <li class="form-group">              
+              <input class="types" type="email" id="email" placeholder="jovasal@gmail.com" required/>
+            </li>
+            <li class="form-group"> 
+             <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            </li>
+            <li class="form-group">
+              <input  value="Get in touch" type="submit" class="btn-input">
+            </li>
```

Comment:
👍 Great to see that you made an effort to improve your input fields based on the previous reviewer's feedback. However, there are still a few adjustments that need to be made:

- [ ] Please ensure that you use proper spacing in your input fields to match the design. Take a look at the design and add the appropriate spacing. 🙏
- [ ] [Optional] In your placeholders, you are currently using "Lorem Ipsum," which is not meaningful. Please refer to the image provided and update the placeholders accordingly to provide more relevant information. 🙏
![Frame 40 (1)](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/55840999/5fbd7b56-17ae-4b69-8b43-b3d0f9f5f1fd)


Taking care of these details will enhance the overall user experience and align your input fields with the design. Keep up the good work! 👍

Reply:
I added the spaces mentioned before. Also, I changed name and email as described in FIGMA

![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/31852a6a-a122-475a-a811-3ea1ed314ceb)